### PR TITLE
Update meteor.js to fix "iron run" command bug on windows

### PR DESCRIPTION
**Pull request by spyhole**
and fixed "Run error on Windows" #138

### CODE REVIEW

File: lib/tools/meteor.js [hunk 1]
```diff
@@ -163,11 +163,12 @@ module.exports.invokeMeteorCommand = function invokeMeteorCommand(cmd, args) {
 
   var future = new Future;
 
-  var child = spawn('meteor', [cmd].concat(args), {
+  var child = spawn(process.platform === "win32" ? 'meteor.bat' : 'meteor', [cmd].concat(args), {
```
Comment (chrisbutler) [inline lib/tools/meteor.js]: what is meteor.bat?

Comment (spyhole) [inline lib/tools/meteor.js]: yeah! @chrisbutler the exec file is "meteor.bat". and you can find it in here. ^_^

https://github.com/meteor/meteor/blob/832e6fe44f3635cae060415d6150c0105f2bf0f6/scripts/admin/publish-meteor-tool.bat

Comment (chrisbutler) [inline lib/tools/meteor.js]: @spyhole i think it's actually this one? https://github.com/meteor/meteor/blob/832e6fe44f3635cae060415d6150c0105f2bf0f6/meteor.bat but nice work!

Comment (spyhole) [inline lib/tools/meteor.js]: @chrisbutler bingo!

**Comment by wodCZ** [comment]
works great now!

**Comment by chrisbutler** [comment]
:+1:

**Pull request merged**